src/components/singleBolgPage/index.css

```diff
@@ -0,0 +1,129 @@
+.articleHeader,
+.authorImage,
+.postInfo {
+  display: flex;
+}
+
+.autherTitle,
+.replyFields h5,
+.recommendedPost h4,
+.postInfo p,
+.publishingDate {
+  font-family: Roboto, Sans-serif;
```

Comment:
is the whole page in roboto? if so, then the parent container should just have this rule

Reply:
No we have two font families, Roboto and Noto Sans